### ISSUE TITLE
About the consistency_weight

### ISSUE THREAD

**Issue by FrankWuuu** [ycwu1997/MC-Net#5]
Hello，
Thanks for sharing your code.
Recently I try to read your code 
If I did not misunderstood  the code, 
then for 2D, the length  of dataloader is 150，the max_iteration is 30000，then the epochs is 200.
That means consistency_weight can only reach 0.1 in your final epoch 200. During training it is less than 0.1 and even less than 0.05 in the first half of the epoch.
So did it really influence the result？the total loss is mostly or even all influenced by the dice loss
I wonder whether if you can just get the best_performance before it works！
Thanks for your reply

**Comment by ycwu1997** [comment]
The selection of consistency weights is the same as the compared methods (See https://github.com/HiLab-git/SSL4MIS/blob/master/code/train_uncertainty_rectified_pyramid_consistency_2D.py). I do not change this part.  By the way, I just checked the training logs and found that the model would converge at the latter training stage, so the consistency loss should have its effectiveness. You can try it via your own setting. Meanwhile, in this paper, all experiments should follow the same experimental setting. Thanks.
